content/news/XXXX-XX-XX-mixxx-2-3-5-released.md

```diff
@@ -0,0 +1,62 @@
+title: Mixxx 2.3.5 released
+authors: Nikolaus Einhauser
+tags: 2.3, 2.3.5, release announcement
+comments: yes
+status: draft
+
+Another bugfix release, this soon?
+
+Yes, unfortunately it turned out [2.3.4 introduced a couple serious
+bugs](https://github.com/mixxxdj/mixxx/issues?q=is%3Aissue+is%3Aclosed+milestone%3A2.3.5+label%3Aregression),
+that required prompt action to fix them and to rectify that sub-par release.
+
+The most serious issue we fixed was a regression that caused an empty waveform
+overview after loading a track. This issue has been fixed in version 2.3.5,
+along with several other bug fixes and improvements.
+
+Other notable changes include requesting microphone and line-in access
+permission on macOS, allowing explicit selection of buffers of 2048 and 4096
+frames/period in the JACK API, making the Beat FX section of the Pioneer DDJ-400
```

Comment:
`period` sounds a bit out-of-place, I feel like that should be plural word too. Not super familiar with the domain-specifics though, so not sure how to rephrase this.

Reply:
IMO "frames per period" is the correct terminology. Just like "frames per second". So I'd keep keep "frames/period" or spell out the "/".

Reply:
Ah okay, I misread that as an 'or', thanks for clarifying!

Reply:
Thank you for the valid criticism, I updated the section to say "frames per period" since this is so end-user facing, the confusion could be more common.